CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# @neo4j/neo4j-genai-python
+
+## 0.2.0
```

Comment:
We don't know the version until the actual release, could you make this "Next" or something instead?